lib/melodica_inventory_web/router.ex

```diff
@@ -48,6 +48,7 @@ defmodule MelodicaInventoryWeb.Router do
     resources "/categories", CategoryController, only: [:edit, :update, :delete, :new, :create]
     resources "/variations", VariationController, only: [:edit, :update, :delete, :new, :create]
     resources "/items", ItemController, only: [:edit, :update, :delete]
+    put "/items/:id/delete_images", ItemController, :delete_images
```

Comment:
The benefit will be sticking to a standard convention, instead of a custom action. In the particular case it seems to me that using REST won't be that hard. The idea is that using standard REST will make the code easier to understand for a new dev coming into the project.